dm/pkg/conn/basedb.go

```diff
@@ -175,6 +177,61 @@ func (d *BaseDB) GetBaseConn(ctx context.Context) (*BaseConn, error) {
 	return baseConn, nil
 }
 
+func (d *BaseDB) ExecContext(tctx *tcontext.Context, query string, args ...interface{}) (sql.Result, error) {
+	if tctx.L().Core().Enabled(zap.DebugLevel) {
+		tctx.L().Debug("exec context",
+			zap.String("query", utils.TruncateString(query, -1)),
+			zap.String("argument", utils.TruncateInterface(args, -1)))
+	}
+	return d.DB.ExecContext(tctx.Ctx, query, args...)
+}
+
+func (d *BaseDB) QueryContext(tctx *tcontext.Context, query string, args ...interface{}) (*sql.Rows, error) {
+	if tctx.L().Core().Enabled(zap.DebugLevel) {
```

Comment:
just a question: why this this check `if tctx.L().Core().Enabled(zap.DebugLevel)`, can we just use `tctx.L().Debug(`?

Reply:
it calls `utils.TruncateString(` twice before `tctx.L().Debug`, it's cheaper to check log level first.